src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -7,6 +17,25 @@
 </head>
```

Comment:
I think you forgot to delete the </head> and <body> here.

You added a new </head> and <body> below in this pull request.